README.md

```diff
@@ -1 +1,90 @@
-# Set-Up-Hello-Microverse-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+# 📖 [Set-up-a-Hello-Microverse-project] <a name="about-project"></a>
+
+**[Set-up-a-Hello-Microverse-project]** is a project in which we will setup linters and html file and change the color of text by using external style.css file.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+I used VS code to create the project and files needed plus used Git and GitHub to complete the tasks.
+
+### Key Features <a name="key-features"></a>
+
+- **[It says Hello to microverse in a green color.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+It will say Hello to microverse in Browser.
+
+### Prerequisites
+
+In order to run this project you need to clone the repo using git bash and run the html file in browser using VS code.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+Bilal Ahmed
+
+👤 **Author1**
+
+- GitHub: [@bilalrajput09](https://github.com/bilalrajput09)
+- Twitter: [@bilal_rajput09](https://twitter.com/bilal_rajput09)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/bilal-ahmed-18b12019a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Search Bar]**
+- [ ] **[ Forms ]**
+- [ ] **[Professional Layout]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Look at what I did !! Your support matters a lot
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving us the chance to build a better world.
+I would like to appreciate my team members.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+
+## 📝 License <a name="License"></a>
+
+This project is [MIT](./License) licensed.
```

Comment:
- [ ]  Jut as the previous reviver mentioned,  your link is still broken here , kindly fix this for your project to be completed
- Consider the fact it's all in a capital letters on your project, then a small case on the link.